src/components/primitive/image/image.component.props.ts

```diff
@@ -1,9 +1,39 @@
-interface ImageComponentProps {
-  src: string;
-  altText: string;
-  className?: string;
+import { Image } from '../../../themes';
+interface ImageComponentProps extends React.ImgHTMLAttributes<HTMLImageElement> {
+  /**
+   * Source for the image
+   */
+  src?: string;
+
+  /**
+   * Fallback text if image fails to load.
+   */
+  altText?: string;
```

Comment:
Is this the equivalent of HTML `alt`? If yes, then we should keep it as alt. Functionality can always be extended but modification must be restricted.